feature.env

```diff
@@ -9,6 +9,7 @@ S3_SECRET_ACCESS_KEY=incentives
 DPS_URL=http://localhost:3000
 NOMIS_USER_ROLES_API_URL=http://localhost:9091/nomisUserRolesApi
 HMPPS_AUTH_URL=http://localhost:9091/auth
+MANAGE_USERS_API_URL=http://localhost:9091/manage-users-api
```

Comment:
this is the same as `NOMIS_USER_ROLES_API_URL`

Reply:
I don't think so, the URL is definitely different for example: 

```
NOMIS_USER_ROLES_API_URL: "https://nomis-user-roles-api-dev.prison.service.justice.gov.uk"
MANAGE_USERS_API_URL: "https://manage-users-api-dev.hmpps.service.justice.gov.uk"
```

I confess these APIs and the way they're named confuses me (but maybe it's just because I'm not too involved into these).

We use the NomisUserRoles API to get the user case loads, while the Manage Users API is now used to get the user and user roles (instead of taking using 2 now deprecated HMPPS Auth API endpoints)

If it makes sense 🙃

Reply:
but should the "manage users" one be used in place of "nomis user roles" one?

Reply:
I've checked the swagger docs and the two APIs are not the same. Specifically, the NOMIS User Roles API endpoint we're using to get the user caseloads (`GET /me/caseloads`) is _**not there**_ in Manage Users API.

So short answer is no, it's not a drop-in replacement and we can't use Manage Users API to do the same thing.

Reply:
long answer and new plan _will_ be:
• the template will be updated to use the token to look up the current user’s roles (we will need to cherry-pick again!)
• we should (maybe) use the prison api to look up a user’s case loads (the NOMIS User Roles api is supposed to be more internal to feed other app-facing apis)